src/pages/tasks/NewTaskDescriptionPage.tsx

```diff
@@ -1,66 +1,58 @@
+import type {StackScreenProps} from '@react-navigation/stack';
 import ExpensiMark from 'expensify-common/lib/ExpensiMark';
-import PropTypes from 'prop-types';
 import React from 'react';
 import {View} from 'react-native';
 import {withOnyx} from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx';
 import FormProvider from '@components/Form/FormProvider';
 import InputWrapperWithRef from '@components/Form/InputWrapper';
+import type {FormInputErrors, FormOnyxValues} from '@components/Form/types';
 import HeaderWithBackButton from '@components/HeaderWithBackButton';
 import ScreenWrapper from '@components/ScreenWrapper';
 import TextInput from '@components/TextInput';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
 import useAutoFocusInput from '@hooks/useAutoFocusInput';
+import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
-import compose from '@libs/compose';
 import * as ErrorUtils from '@libs/ErrorUtils';
 import Navigation from '@libs/Navigation/Navigation';
+import type {NewTaskNavigatorParamList} from '@libs/Navigation/types';
 import updateMultilineInputRange from '@libs/updateMultilineInputRange';
-import * as Task from '@userActions/Task';
+import * as TaskActions from '@userActions/Task';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
 import INPUT_IDS from '@src/types/form/NewTaskForm';
+import type {Task} from '@src/types/onyx';
 
-const propTypes = {
-    /** Grab the Share description of the Task */
-    task: PropTypes.shape({
-        /** Description of the Task */
-        description: PropTypes.string,
-    }),
-
-    ...withLocalizePropTypes,
+type NewTaskDescriptionPageOnyxProps = {
+    /** Grab the Share title of the Task */
```

Comment:
NAB/NIT: In this comment, `Share title` is kinda irrelevant to this description page. Maybe just `Details of the Task` or `Grab the details of the Task` appears better to me. 

Or just omit this comment? Does not appear to be useful at all.

Same for the `NewTaskDetailsPage` and `NewTaskTitlePage`. I think `Share` is only applicable to `Share somewhere` and is not for details and title.

Reply:
updated